tests/api/test_scalars.py

```diff
@@ -0,0 +1,44 @@
+from tests.api.test_checkout import QUERY_CHECKOUT
```

Comment:
I thin we can duplicate the query here. In the near feature, we want to move tests next to code and it can be not good practice if someone changes mutation in checkout test and gets errors in scalar tests.

Reply:
Done